packages/jellyfish-api-core/__tests__/category/loan/listVaults.test.ts

```diff
@@ -209,9 +209,34 @@ describe('Loan listVaults with options and pagination', () => {
     })
     await testing.generate(1)
 
-    await testing.token.mint({
-      symbol: 'AAPL',
-      amount: 10
+    // setup loan token vault for loaning of loan token
+    const loanTokenSchemeId = 'borrow'
+    await testing.rpc.loan.createLoanScheme({
+      minColRatio: 100,
+      interestRate: new BigNumber(0.01),
+      id: loanTokenSchemeId
+    })
+    await testing.generate(1)
+    const loanVaultOwner = await testing.generateAddress()
+    const loanTokenVaultId = await testing.rpc.loan.createVault({
+      ownerAddress: loanVaultOwner,
+      loanSchemeId: loanTokenSchemeId
+    })
+    await testing.generate(20)
```

Comment:
huh?? is this by intentional??

Reply:
hmm actually nothing to do with gen(20) here.. so better change back to gen(1) to avoid confusion